base/applications/mspaint/precomp.h

```diff
@@ -21,8 +21,9 @@
 #include <shellapi.h>
 #include <htmlhelp.h>
 
+#define NO_STRSAFE
```

Comment:
What's the problem with using stringsafe here?

Reply:
The `<strsafe.h>` implementation can refuse incompatible function calls.

https://github.com/reactos/reactos/blob/47f6745bcdfb833b0311e052f04e6cd5634660eb/sdk/include/psdk/strsafe.h#L1921

So, some programs are incompatible to `<strsafe.h>`.